public/app/features/explore/utils/decorators.test.ts

```diff
@@ -84,6 +84,7 @@ const createExplorePanelData = (args: Partial<ExplorePanelData>): ExplorePanelDa
     tableResult: null,
     traceFrames: [],
     nodeGraphFrames: [],
+    customFrames: [],
```

Comment:
I know it's not done for all the cases but it'd be good to set some customFrames here to have and example and a test coverage.

Reply:
I have added a test, but did you want it explicitly in this method as well?

Reply:
Looks good, I think 4cdbf013fd10927a7c4dbf3c48c1180abfbd5414 covers it well 👍